src/utils.js

```diff
@@ -109,7 +109,10 @@ export function appendTrailingSlash(str) {
 }
 
 export function log(...params) {
-  if (process.env.NODE_ENV !== 'production') {
+  if (
+    process.env.NODE_ENV !== 'production' &&
+    process.env.NODE_ENV !== 'test'
```

Comment:
We have a few tests (routing) that are causing the logger to emit statements during tests, so this turns them off.